src/main/java/medic/gateway/alert/SettingsStore.java

```diff
@@ -67,19 +69,21 @@ public static SettingsStore in(Context ctx) {
 	public static final Pattern URL_PATTERN = Pattern.compile(
 			"http[s]?://([^/:]*)(:\\d*)?(.*)");
 
-	public static final long POLL_INTERVAL = 30 * 1000L;
+	public static final long POLL_INTERVAL = 3 * 60 * 1000L; // 3 min in milliseconds
```

Comment:
What made you pick 3 minutes? Is 30 seconds too frequent?

Reply:
Yes, I think 30 seconds is too frequent and potentially warn the phone system that there's some app being too active in background (?) I wanted to give it some "air" and see if that helps with the freezing thing in 1.5.X and 1.7.X.
While 3 minutes sounds reasonable, not too fast not too slow... Considering that Asia team said that in a very busy day each phone will receive a max of 200 sms per day and our batch is size 10, so it will take 60 min tops if we receive all 200 sms at once.
If this is too long, what about if we decrease it to 2 min?

Reply:
Funny thing I'm seeing: While checking the logs, even though the code is set to 30 seconds, the app is logging request to Webapp every 45 to 60 seconds. There's some delay in the logging.

Reply:
Related: https://github.com/medic/medic-gateway/issues/140

I think the poll interval has been 30 seconds for ages so it doesn't explain why 1.7x would be failing, does it? I'd rather not change anything if we're not confident it'll help.

Reply:
Oh that one is because we also request to Webapp [here](https://github.com/medic/medic-gateway/blob/37c51f742691485ef59117e259773432006bcd45/src/main/java/medic/gateway/alert/IntentProcessor.java#L94), when there's a `SMS_DELIVER_ACTION` from the intent, then it triggers the requests to Webapp. 
This is out of the flow from the Alarm/WakeService that has the interval.

Reply:
This change has been reverted